packages/starksheet-webapp/src/components/Footer/Footer.tsx

```diff
@@ -4,7 +4,7 @@ import { useContext } from "react";
 import { CELL_BORDER_WIDTH, CELL_HEIGHT, N_ROW } from "../../config";
 import { AccountContext } from "../../contexts/AccountContext";
 import { OnsheetContext } from "../../contexts/OnsheetContext";
-import { chainProvider } from "../../provider";
+import { useChainProvider } from "../../hooks/useChainProvider";
```

Comment:
I created a useChainProvider hook in this PR to centralize the declaration logic (see file below).